pkg/cmd/pkg/package_list_test.go

```diff
@@ -58,6 +58,59 @@ func TestListPackages(t *testing.T) {
 `, out.String())
 }
 
+var testCases = []struct {
+	cluster string
+	options listOptions
+	out     string
+}{
+	{NoPlugins, listOptions{jsonOutput: true},
+		`[{"apps":["a","b","some-app"],"description":"package-2","framework":false,"name":"package-1","selected":false,"version":"0.0.0.1"},
```

Comment:
What do you think about instead of using string literals for the json response, what do you think about using cosmos.Package like how it's done on line 29? It'd be clearer what the json is, and it'd be easier to read.

Reply:
Line 29 is for mocking cosmos resposne. This JSON string is the actual output from CLI. I can change it to a struct but then it will be converted to JSON.

Reply:
Ya fair enough. So do you prefer to keep it as a string literal?

Reply:
Fixed